### PR TITLE
Updated DiezAlamo-2018-750

### PR DESCRIPTION
**Pull request by mathildavz**
I've mapped some unlinked concepts in DiezAlamo-2018-750.

### CODE REVIEW

**Comment by mathildavz** [comment]
I will add some comments today in the afternoon.

**Comment by mathildavz** [comment]
- DiezAlamo-2018-750-30 - ELEPHANT TRUNK: I think the piece of furniture is meant, I would unmap
- DiezAlamo-2018-750-78 - HAWK: the toy kite is meant, I would unmap
- DiezAlamo-2018-750-83 - CROWN OF HEAD: it seems to refer to money, I would unmap
- DiezAlamo-2018-750-127 - LEAF (LEAFLIKE OBJECT): the word refers to reading according to the comments
- DiezAlamo-2018-750-173 - PALM TREE: a unit for distance seems to be meant, I would unmapped
- DiezAlamo-2018-750-196 - RADIO: chemical element is meant, I would unmap
- DiezAlamo-2018-750-529 - POT: they mean a carpenter's tool, I would unmap

**Comment by AnnikaTjuka** [comment]
Another list with Spanish glosses. @MuffinLinwist Could you also review the list? I'll send you some screenshots on how to make a review with multiple comments via Mattermost.

**Comment by LinguList** [comment]
Yes, thanks in advance for quickly checking, @MuffinLinwist, I can then check after your comments.

**Comment by MuffinLinwist** [comment]
Yes, @AnnikaTjuka, @mathildavz, and @LinguList. I am starting to work on this.

**Review by MuffinLinwist — COMMENTED** [review]
@AnnikaTjuka, @mathildavz, and @LinguList my review is done.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 1]
```diff
@@ -64,37 +64,37 @@ DiezAlamo-2018-750-62	62	CERDO	pig	PIG	1337	4.21212121212121	6.92	2.833333333333
...
 DiezAlamo-2018-750-72	72	CIGARRO	cigarette	CIGARETTE	969	2.78125	3.88	7.83333333333333	5.30769230769231	1.48387096774194	2.6875	7.36	6.01694915254237	6.247	6.308	
 DiezAlamo-2018-750-73	73	CINTURÓN	belt	BELT	772	4.09375	2.56	7.16666666666667	4.65384615384615	2.74193548387097	2.3125	2.72	6.13559322033898	6.051	5.833	Prendas de vestir (An article of clothing)
 DiezAlamo-2018-750-74	74	COCINA	kitchen	KITCHEN	1365	4.21875	1.28	1.33333333333333	3.69230769230769	4.83870967741935	2.75	4.76	4.55	6.044	6.059	Partes de un edificio (A part of a building)
 DiezAlamo-2018-750-75	75	COHETE	rocket			5.78787878787879	6.24	2.33333333333333	5.38461538461539	7.19354838709677	2.15625	3.92	3.56140350877193	5.952	6.271	Vehículos (A type of vehicle)
-DiezAlamo-2018-750-76	76	COJÍN	cushion			5.3030303030303	2.28	7.29166666666667	1.76923076923077	1.41935483870968	2.5625	2.84	6.01694915254237	6.022	6.429	Muebles (An article of furniture)
+DiezAlamo-2018-750-76	76	COJÍN	cushion	PILLOW	1044	5.3030303030303	2.28	7.29166666666667	1.76923076923077	1.41935483870968	2.5625	2.84	6.01694915254237	6.022	6.429	Muebles (An article of furniture)
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:77]: In this case, I'd unmap.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 2]
```diff
@@ -115,7 +115,7 @@ DiezAlamo-2018-750-113	113	FOCO	spotlight			5.8125	2.2	4.20833333333333	3.192307
 DiezAlamo-2018-750-114	114	FOLIO	sheet			2.90625	2.6	7.875	2.38461538461538	1.48387096774194	2.6875	2.92	6.12280701754386	6.574	7	Material de lectura (A type of reading material)
 DiezAlamo-2018-750-115	115	FOTO	photo	PICTURE (PHOTOGRAPH)	3304	6.61290322580645	1.56	7.625	2.15384615384615	1.38709677419355	2.9375	2.12	5.53333333333333	6.152	6.436	
 DiezAlamo-2018-750-116	116	FRESA	strawberry	STRAWBERRY	3138	7.125	2.28	7.70833333333333	1.30769230769231	1.2258064516129	7.71875	5.52	6.06779661016949	5.878	6.67	Frutas (A fruit)
-DiezAlamo-2018-750-117	117	FUSIL	rifle			3.875	3.56	4.58333333333333	7.88461538461539	6.67741935483871	1.5625	2.8	4.8135593220339	5.909	5.836	Armas (A weapon)
+DiezAlamo-2018-750-117	117	FUSIL	rifle	GUN	1566	3.875	3.56	4.58333333333333	7.88461538461539	6.67741935483871	1.5625	2.8	4.8135593220339	5.909	5.836	Armas (A weapon)
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:118]: There are some 'rifle' glosses in other lists that are currently unmapped. @mathildavz Could you open an issue where we discuss adding a new concept RIFLE?

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 3]
```diff
@@ -125,15 +125,15 @@ DiezAlamo-2018-750-123	123	HAMACA	hammock	HAMMOCK	2019	4	4.48	4.25	2.15384615384
 DiezAlamo-2018-750-124	124	HIERBA	grass	GRASS	606	6.5625	3.29166666666667	6.54166666666667	1.26923076923077	1.51612903225806	4.09375	5.56	5.23728813559322	6.536	6.087	
 DiezAlamo-2018-750-125	125	HIERRO	iron	IRON	621	4.1875	1.72	5.79166666666667	5.5	4.06451612903226	2.03125	3.72	5.23728813559322	5.849	5.573	Metales (A metal)
 DiezAlamo-2018-750-126	126	HILO	thread	THREAD	1161	3.4375	2.8	7.625	1.88461538461538	1.09677419354839	3.15625	1.16	5.55172413793103	5.825	6.37	Tejidos (A kind of cloth)
-DiezAlamo-2018-750-127	127	HOJA	leaf			3.72727272727273	3.68	7.66666666666667	2.15384615384615	1.48387096774194	3.25	2.96	6.2	5.261	6.232	Material de lectura (A type of reading material)
+DiezAlamo-2018-750-127	127	HOJA	leaf	LEAF (LEAFLIKE OBJECT)	2250	3.72727272727273	3.68	7.66666666666667	2.15384615384615	1.48387096774194	3.25	2.96	6.2	5.261	6.232	Material de lectura (A type of reading material)
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:128]: Yes, the concept LEAF (LEAFLIKE OBJECT) excludes the "plant meaning". In other lists, we mapped 'hoja' to LEAF so I'd suggest to do the same here.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 4]
```diff
@@ -171,13 +171,13 @@ DiezAlamo-2018-750-169	169	OLLA	pot	POT	1462	3.57575757575758	2.32	6.13043478260
 DiezAlamo-2018-750-170	170	ÓRGANO	organ			5.24242424242424	3.88	3.625	3.34615384615385	4.35483870967742	2.4375	4.16	4.03448275862069	4.767	5.587	Instrumentos musicales (A musical instrument)
 DiezAlamo-2018-750-171	171	OSO	bear	BEAR	1820	4.75	6.88	1.29166666666667	6.19230769230769	6.12903225806452	4.3125	6.08	4.05084745762712	6.383	5.342	Animales de cuatro patas (A four-footed animal)
 DiezAlamo-2018-750-172	172	OVEJA	sheep	SHEEP	1331	3.15151515151515	6.76	2.625	2.73076923076923	4.12903225806452	5.4375	6.2	5.06779661016949	6.207	6.322	Animales de cuatro patas (A four-footed animal)
-DiezAlamo-2018-750-173	173	PALMA	palm			3.24242424242424	4.08	5.125	3.26923076923077	3.45161290322581	2.9375	2.92	5.1864406779661	4.05	6.417	Unidades de longitud (A unit of distance)
+DiezAlamo-2018-750-173	173	PALMA	palm	PALM TREE	1181	3.24242424242424	4.08	5.125	3.26923076923077	3.45161290322581	2.9375	2.92	5.1864406779661	4.05	6.417	Unidades de longitud (A unit of distance)
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:174]: The category `Unidades de longitud (A unit of distance)` indicates that the concept 1183 PALM OF HAND is meant.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 5]
```diff
@@ -688,7 +688,7 @@ DiezAlamo-2018-750-686	686	DOMICILIO	home	HOME	2406	2.54166666666667	1.576923076
...
 DiezAlamo-2018-750-689	689	SANTUARIO	sanctuary			3.875	1.46153846153846	1.11538461538462	1.83333333333333	2.33333333333333	2	3.375	3.27586206896552	5.897	5.807	Edificios religiosos (A building for religious service)
-DiezAlamo-2018-750-690	690	VÍA	track			2.875	2.53846153846154	2.07692307692308	2.79166666666667	4.83333333333333	1.91666666666667	2.41666666666667	3.27118644067797	3.914	5.802	
+DiezAlamo-2018-750-690	690	VÍA	track	FOOTPRINT	318	2.875	2.53846153846154	2.07692307692308	2.79166666666667	4.83333333333333	1.91666666666667	2.41666666666667	3.27118644067797	3.914	5.802	
 DiezAlamo-2018-750-691	691	ANTÍDOTO	antidote			3.75	2.96153846153846	6.23076923076923	2	1.16666666666667	4.125	2.79166666666667	3.25862068965517	4.917	3.583	
 DiezAlamo-2018-750-692	692	BUQUE	vessel			3.45833333333333	5.15384615384615	1.61538461538462	3.5	5.5	1.79166666666667	2.625	3.25423728813559	5.677	5.712	Embarcaciones (A type of ship)
 DiezAlamo-2018-750-693	693	COMERCIO	shop	SHOP	746	4.29166666666667	2.34615384615385	1.34615384615385	1.58333333333333	4.75	2.29166666666667	3.25	3.23728813559322	5.14	5.323
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:694]: The authors state that the list represents concrete concepts. So I wouldn't map to a verb concept.

**Comment by AnnikaTjuka** [comment]
Many thanks for the review, @MuffinLinwist! I left a few comments, but I mostly agree with your suggestions.

**Comment by LinguList** [comment]
I just quickly checked the discussion here. If you need me to have a thorougher look, this may have to wait until next week. Otherwise, I agree and approve with the discussion and the then final version.

**Review by LinguList — APPROVED** [review]

**Comment by mathildavz** [comment]
I've implemented the changes. Does someone want to check again, or should I just merge?

**Comment by AnnikaTjuka** [comment]
I'll have a look at the changes today.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv (hunk 3 above)
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:128]: This needs to be mapped to LEAF.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:118]: I'd unmap this for now.

File: concepticondata/conceptlists/DiezAlamo-2018-750.tsv [hunk 6]
```diff
@@ -549,12 +549,12 @@ DiezAlamo-2018-750-547	547	CORNEJA	raven	RAVEN	2543	3.79166666666667	4.4	3.79166
 DiezAlamo-2018-750-548	548	CUCHILLA	razor	RAZOR	151	2.875	3	7.33333333333333	6.64	2.75	1.56	2.16666666666667		5.815	5.459	Armas (A weapon)
 DiezAlamo-2018-750-549	549	REVÓLVER	revolver			3.79166666666667	3.68	6.83333333333333	7.6	6.29166666666667	1.8	3.08333333333333		6.141	6.262	Armas (A weapon)
 DiezAlamo-2018-750-550	550	RUIBARBO	rhubarb			3.79166666666667	4.2	4.16666666666667	2.48	2.25	3.64	3.58333333333333				
-DiezAlamo-2018-750-551	551	RIFLE	rifle			3.04166666666667	3.72	5.79166666666667	7.4	6.625	1.76	2.83333333333333				Armas (A weapon)
+DiezAlamo-2018-750-551	551	RIFLE	rifle	GUN	1566	3.04166666666667	3.72	5.79166666666667	7.4	6.625	1.76	2.83333333333333				Armas (A weapon)
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/DiezAlamo-2018-750.tsv:552]: I'd unmap it for now.

**Review by AnnikaTjuka — CHANGES_REQUESTED** [review]
There are still a few unresolved issues. I added comments accordingly.

**Comment by LinguList** [comment]
Yes, @AnnikaTjuka, hoja should be just mapped to LEAF.

**Comment by mathildavz** [comment]
Sorry for missing those! I have implemented the comments now, and it should be ready to merge. @AnnikaTjuka

**Review by AnnikaTjuka — APPROVED** [review]
Great! Now everything looks good and you can merge, @mathildavz.

**Pull request merged**